elastic.go

```diff
@@ -396,36 +427,86 @@ func cmdNode(c *cli.Context, subCmd string) string {
 	default:
 		route = ""
 	}
-	return url + route
+	return route
 }
 
 func cmdQuery(c *cli.Context) string {
 	route := c.Args().First()
-	url := c.GlobalString("baseurl")
-	return url + route
+	return route
 }
 
 func cmdStats(c *cli.Context, subCmd string) string {
 	var route string
-	url := c.GlobalString("baseurl")
+
 	switch subCmd {
 	case "size":
-		route = "_stats/index,store"
+		route = "_stats/indexing,store"
 	default:
 		route = ""
 	}
-	return url + route
+	return route
 }
 
-func httpGet(route string, trace bool) (*http.Response, error) {
-	if trace {
-		fmt.Fprintf(os.Stderr, "GET: %s", route)
+func httpGetAll(route string, c *cli.Context) ([]*http.Response, error) {
```

Comment:
This function will get the results from all the ES nodes. It will be used for commands like `elastic node status`. What I am not sure about it is how to deal with a node down.